src/managedwriter/json_writer.ts

```diff
@@ -0,0 +1,124 @@
+// Copyright 2023 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import {protobuf} from 'google-gax';
+import * as protos from '../../protos/protos';
+import {PendingWrite} from './pending_write';
+import {StreamConnection, RemoveListener} from './stream_connection';
+import * as adapt from '../adapt';
+import {Writer} from './writer';
+
+type TableSchema = protos.google.cloud.bigquery.storage.v1.ITableSchema;
```

Comment:
I think you could just import the types directly vs., redeclaring a `type`:

```
import {ITableSchema} from `../../protos/protos/google/cloud/bigquery/storabge/v1`
import {IDescriptorProto, DescriptorProto} from '../../protos/protos/
```

And it might feel a bit more idiomatic (curious to hear @alexander-fenster's thoughts).

Reply:
That may be more idiomatic, but importing and then customizing type name for the classes is the norm for the BQ Node libraries at the moment, as is not using "I" types directly in the code for the class.